controls/SV-204428.rb

```diff
@@ -1,32 +1,32 @@
 control 'SV-204428' do
-  title 'The Red Hat Enterprise Linux operating system must lock the associated account after three unsuccessful
-    root logon attempts are made within a 15-minute period.'
+  title "The Red Hat Enterprise Linux operating system must lock the associated account after #{input('unsuccessful_attempts')} unsuccessful
+    root logon attempts are made within a #{input('fail_interval')/60}-minute period.'
   desc 'By limiting the number of failed logon attempts, the risk of unauthorized system access via user password
-    guessing, otherwise known as brute forcing, is reduced. Limits are imposed by locking the account.'
-  desc 'check', 'Verify the operating system automatically locks the root account, for a minimum of 15 minutes, when
-    three unsuccessful logon attempts in 15 minutes are made.
+    guessing, otherwise known as brute forcing, is reduced. Limits are imposed by locking the account."
+  desc 'check', "Verify the operating system automatically locks the root account, for a minimum of #{input('fail_interval')/60} minutes, when
+    #{input('unsuccessful_attempts')} unsuccessful logon attempts in #{input('fail_interval')/60} minutes are made.
     # grep pam_faillock.so /etc/pam.d/password-auth
-    auth required pam_faillock.so preauth silent audit deny=3 even_deny_root fail_interval=900 unlock_time=900
-    auth [default=die] pam_faillock.so authfail audit deny=3 even_deny_root fail_interval=900 unlock_time=900
+    auth required pam_faillock.so preauth silent audit deny=#{input('unsuccessful_attempts')}even_deny_root fail_interval=#{input('fail_interval')} unlock_time=#{input('fail_interval')}
```

Comment:
Space needed before `even_deny_root`

Reply:
`fail_interval` input was used for both fail_interval and unlock_time values. Corrected it along with other formatting updates.